nav2_toolkit/config/last_known_pose.yaml

```diff
@@ -0,0 +1,10 @@
+pose:
```

Comment:
Why is this file included? Remove please

Reply:
Also, this should be the stamped version, we are going to want to also store the header frame ID and the time it was saved. We probably want to override the timestamp later, but it would be good to know when the last one was from.